community/kernel/src/main/java/org/neo4j/kernel/impl/api/KernelTransactionImplementation.java

```diff
@@ -484,6 +484,10 @@ public void assertOpen()
         {
             throw new TransactionTerminatedException( reason );
         }
+        if ( closed )
```

Comment:
when can this happen? I can't find any test for this state (maybe I miss that in a group of renamings and stuff so please point me into that)